### ISSUE TITLE
[Lighthouse / Offscreen Images / Helpful]

### ISSUE THREAD

**Issue by sperks38** [google/WebFundamentals#4701]
While I appreciate the suggestion of lazy-loading images, I feel that not mentioning consideration for accessibility (even if you consider loading an image via JS no longer an issue) is questionable, especially for a company that many see as the primary source for SEO/accessibility.

**Comment by ebidel** [comment]
Can you expand a bit on "mentioning consideration for accessibility"? Is there a specific scenario you're thinking of?

ATM, Lighthouse only considers things that happen during page load performance. What happens after that period is the responsibility of the site. As an example, if a site immediately loads below-the-fold images after the page load, a11y doesn't suffer. Assistive technologies should be fine with that.

cc @robdodson for his expertise.

**Comment by sperks38** [comment]
Eric,

Sorry it took me a little time to follow up on this (esp considering how quickly you responded to me).

The gist of my comment here is centered around the usage of the term "lazy loading" and I incorrectly assumed scroll to be involved. I realize this isn't what you mean, but I feel the term 'lazy load' does infer infinite scroll which does raise SEO/accessibility concerns.

https://www.newfangled.com/pros-and-cons-of-infinite-scrolling-content/

I'm actually glad that you brought this recommendation up, as I feel it's a practice that certain could be used more even on large sites.

Thanks,
Steve
